packages/next/src/build/webpack/loaders/next-flight-loader/index.ts

```diff
@@ -147,20 +146,7 @@ ${JSON.stringify(ref)},
         }
       }
 
-      const compilation = this._compilation!
-      const originalSourceURL = ModuleFilenameHelpers.createFilename(
-        module,
-        {
-          moduleFilenameTemplate:
-            'webpack://[namespace]/[resource-path]/__nextjs-internal-proxy.mjs',
-          namespace: '_N_E',
-        },
-        {
-          requestShortener: compilation.requestShortener,
-          chunkGraph: compilation.chunkGraph,
-          hashFunction: compilation.outputOptions.hashFunction,
-        }
-      )
+      const originalSourceURL = `webpack://_N_E/${this.utils.contextify(this.context || this.rootContext, this.resourcePath)}/__nextjs-internal-proxy.mjs`
```

Comment:
I'm not comfortable with this for Webpack. I used `ModuleFilenameHelpers.createFilename` because that's what Webpack is using in their devtool plugins.

Can we only use this impl when Rspack is used?